src/Compilers/CSharp/Portable/Symbols/Extensions/SourceExtensionImplementationMethodSymbol.cs

```diff
@@ -65,6 +65,19 @@ internal override int ParameterCount
 
         internal sealed override void AddSynthesizedAttributes(PEModuleBuilder moduleBuilder, ref ArrayBuilder<CSharpAttributeData> attributes)
         {
+            if (UnderlyingMethod is SourcePropertyAccessorSymbol { AssociatedSymbol: SourcePropertySymbolBase extensionProperty })
+            {
+                var priority = extensionProperty.OverloadResolutionPriority;
+                if (priority != 0)
```

Comment:
>priority != 0 [](http://example.com/codeflow?start=20&length=13)

Should the user applied attribute (if there is one) from the property be copied unconditionally?